iosApp/iosApp/ViewModels/StopDetailsViewModel.swift

```diff
@@ -50,6 +59,17 @@ class StopDetailsViewModel: ObservableObject {
         }
     }
 
+    func clearTripDetails() {
```

Comment:
Added tests for the stop details VM, maybe should have split that into a new PR since it bumped us up another 500+ lines and included a bit of refactoring of some mock VMs 😬 While writing tests I realized that I could pull even more code (for error banner stuff) from `StopDetailsPageHandlerExtension` into the VM, but restrained myself.

Reply:
Awesome! Splitting out refactor changes is a good idea to keep in mind for the future 👍